content/en/Getting started/Assets/asset-description.md

```diff
@@ -87,9 +87,9 @@ Make sure to include the:
 - Unique users / roles
 - Applicable network / architecture diagrams
 
-## Asset Documentation
+## Attachments
 
-To share more about your assets, you can upload the documentation of your choice. Our app accepts files
+To share more about your assets, you can upload the documentation of your choice under **Attachment(s)**. Our app accepts files
```

Comment:
Oh you're kidding.... so the app includes the parentheses... (Might have been my mistake)